src/shared/services/map-location/index.js

```diff
@@ -63,29 +46,46 @@ export const mapLocation = loc => {
   return value;
 };
 
-export const formatMapLocation = loc => {
+const getAddressText = ({ openbare_ruimte, huisnummer, huisnummer_toevoeging, postcode, woonplaats }) =>
+  [
+    [openbare_ruimte, huisnummer, huisnummer_toevoeging && `-${huisnummer_toevoeging}`],
```

Comment:
Wouldn't we get here between huisnummer and huisnummer_toevoeging an extra space?

Reply:
I don't think so. Why do think that?

Reply:
I see the unittest  'should render the address name' is failing

Reply:
Ah, I see what you mean. I'm asking around what the correct notation of an address should be.

Reply:
Fixed